apps/vanilla/src/global/variables.css

```
@@ -5,6 +5,7 @@
 	--on-surface-color: rgb(0 0 0 / 100%);
 	--on-surface-dim-color: rgb(0 0 0 / 50%);
 	--primary-color: rgb(241 89 36 / 100%);
+	--winner-color: rgb(16 227 100/50%);
```

Let's keep consistent formatting 
```suggestion
	--winner-color: rgb(16 227 100 / 50%);
```